src/ui/block-timeBased.js

```diff
@@ -17,15 +17,13 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         model: {
             mediaUrl: null,
             alternativeText: null,
-            description: null,
-            transcript: null,
-            hasTranscript: true
```

Comment:
The value of `hasTranscript` is still expected in `themes/base/templates/storyBlockMediaEdit.handlebars` file.